doc/modules/ROOT/pages/python-runtime/knowledge-graph-embeddings.adoc

```diff
@@ -0,0 +1,283 @@
+= Knowledge graph embeddings
+
+Knowledge Graph Embeddings (KGE) refer to a family of algorithms designed to learn low-dimensional representations of entities and relations within a knowledge graph.
+These embeddings are utilized for tasks including link prediction, entity classification, and entity clustering.
+
+This chapter provides an overview of the available shallow embedding algorithms in the GDS Python runtime, such as `TransE` and `DistMult`.
+
+* `TransE`
+** https://proceedings.neurips.cc/paper_files/paper/2013/file/1cecc7a77928ca8133fa24680a88d2f9-Paper.pdf[Bordes, Antoine, et al. "Translating embeddings for modeling multi-relational data." Advances in neural information processing systems 26 (2013).^]
```

Comment:
I think we have said in the past that we should not have links to papers as they may be broken. Instead just state the name of the authors and the paper